t/plugin/proxy-rewrite3.t

```diff
@@ -151,3 +151,57 @@ plugin_proxy_rewrite get method: GET
 --- response_body
 property "method" validation failed: matches none of the enum values
 done
+
+
+
+=== TEST 6: set route(rewrite method with headers)
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                 ngx.HTTP_PUT,
+                 [[{
+                        "methods": ["GET"],
+                        "plugins": {
+                            "proxy-rewrite": {
+                                "uri": "/uri/plugin_proxy_rewrite",
+                                "method": "POST",
+                                "scheme": "http",
+                                "host": "apisix.iresty.com",
+                                "headers":{
+                                    "x-api-version":"v1"
+                                }
+                            }
+                        },
+                        "upstream": {
+                            "nodes": {
+                                "127.0.0.1:1980": 1
+                            },
+                            "type": "roundrobin"
+                        },
+                        "uri": "/hello"
+                }]]
+                )
+
+            if code >= 300 then
+                ngx.status = code
+            end
+            ngx.say(body)
+        }
+    }
+--- response_body
+passed
+
+
+
+=== TEST 7: hit route(with out header)
+--- request
+GET /hello HTTP/1.1
+--- response_body
+uri: /uri/plugin_proxy_rewrite
+host: apisix.iresty.com
+x-api-version: v1
+x-real-ip: 127.0.0.1
+--- grep_error_log_out
```

Comment:
```suggestion
--- error_log
```
Now I know why the test can pass.

Reply:
ok, modified 'TEST 2' and 'TEST 4' case too